plugin/mlsdbonlinelocator.h

```diff
@@ -39,6 +39,9 @@ class MlsdbOnlineLocator : public QObject
     explicit MlsdbOnlineLocator(QObject *parent = 0);
     ~MlsdbOnlineLocator();
 
+    void setWlanDataAllowed(bool allowed);
+    void setWlanSignalStrengthAllowed(bool allowed);
```

Comment:
Provide way to read the value also?

Reply:
removed signal strength stuff but done for wlanDataAllowed